src/core/Entry.cpp

```diff
@@ -620,13 +620,20 @@ QString Entry::resolvePlaceholders(const QString& str) const
 {
```

Comment:
This has been here before, but please rename this method to `resolvePlaceholder()`. Using the plural is just wrong because it doesn't reflect what this method is doing. It only gets a single placeholder and replaces it with a value. It does not take a string full of different placeholders and resolves them as the current name suggests Took me a long time to reverse engineer what this method is actually doing and where it gets its input from in the context of autotype. A better name would have made things clearer from the get.go.